src/database/models/rating.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable require-jsdoc */
+/* eslint-disable valid-jsdoc */
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Rating extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      Rating.belongsTo(models.Accomodation, {
+        foreignKey: 'accomodation_id',
+        onDelete: 'CASCADE'
+      });
+    }
+  }
+  Rating.init({
+    accomodation_id: {
+      type: DataTypes.INTEGER,
+    allowNull: false
+},
```

Comment:
I think you should come up with a logic to add and retrieve ratings based on their categories (numbers 1-5) for a given accommodation, just the number of travelers who rated with a given number between 1 and 5 otherwise it won't be worth to retrieve ratings in this way. Please work on it

Reply:
I have done you can review it again